test/testcafe/spec/DeviceChallengePollView_spec.js

```diff
@@ -317,6 +360,23 @@ test
     await t.expect(identityPage.getIdentifierValue()).eql('Test Identifier');
   });
 
+test
+  .requestHooks(loopbackPollTimeoutMockLogger, loopbackPollTimeoutMock)('new poll does not starts until last one is ended', async t => {
+    await setup(t);
+    // This test verify if new /poll calls are made only if the previous one was finished instead of polling with fixed interval.
+    // Updating /poll response to take 5 sec to response.
+    // Then counting the number of calls that should be done in time interval. Default Timeout for /poll is 2 sec.
+    // Expecting to get only 2 calls(first at 2nd sec, second at 9th(5 sec response + 2 sec timeout) second).
+    const tetsTimeout = 10_000;
```

Comment:
what is `10_000` is it valid ?

Reply:
It needed to verify that at least 2 /poll calls with timeout were made. this allows verifying that the new poll call would not be started before the previous finished. 
The first call is made with a timeout of 2 sec, the second call will be made at the 9th second(2sec timeout of first calls + 2 sec timeout of second call + 5 seconds of first call response). And I decided to add an additional 1 sec to make sure this test is not flaky.
Before updating the logic we would get 4-5 calls(one call per 2 sec). 

Please advise, from your perspective, Is there a more efficient way to check it ?